cmd/massastation/main.go

```diff
@@ -21,18 +21,15 @@ func ParseFlags() api.StartServerFlags {
 
 	_, err := config.GetConfigDir()
 	if err != nil {
-		log.Fatalln("Unable to read config dir:", err,
-			`MassaStation can't run without a config directory.
-			Please reinstall MassaStation using the installer at https://github.com/massalabs/station and try again.`,
-		)
+		config.Logger.Error(fmt.Sprintf("Unable to read config dir: %s\n%s", err, "MassaStation can't run without a config directory.\nPlease reinstall MassaStation using the installer at https://github.com/massalabs/station and try again."))
 	}
 
 	certDir, err := config.GetCertDir()
 	if err != nil {
-		log.Fatalln("Unable to read cert dir:", err,
+		config.Logger.Error(fmt.Sprintf("Unable to read cert dir:%s\n%s", err,
```

Comment:
i think this is not iso functionnal. Fatalln will exit the app. and i dont think config.Logger.Error wil do the same.
or you keep log.Fatalln here. or you have to add a  os.Exit(1) after this line